Do not load sidebar details until they are opened (#20848)
directus/directus: .changeset/thick-spiders-share.md

---
'@directus/app': minor
---

> Do not load sidebar details (revisions, comments, shares etc) until they are opened

Reviewer: Usually past tense is preferred:
```suggestion
Prevented loading of sidebar details (revisions, comments, shares etc) until they are opened
```

Author: Still getting used to that 😅 , fixed!
